app/(admin)/admin/notifications/page.tsx

```diff
@@ -42,30 +47,87 @@ const AdminNotifications = (): JSX.Element => {
     await sendEmailNotifications({ content, groupUsers, subject });
   };
 
+  /**
+   * Function to handle radio selection logic.
+   * @param value - Value of the radio buttons.
+   */
+  const handleRadioChange = (value: string): void => {
+    if (value === 'all') {
+      getLeagueData();
+    }
+  };
+
   return (
-    <section data-testid="admin-notifications-content">
-      <Button
-        label="Email Testers"
-        type="button"
-        onClick={getLeagueData}
-        data-testid="email-testers"
-      />
-      <form onSubmit={handleSubmit}>
-        <Label htmlFor="subject">Subject:</Label>
-        <Input
-          type="text"
-          id="subject"
-          data-testid="subject-text"
-          onChange={(e) => setSubject(e.target.value)}
-        />
-        <Label htmlFor="content">Content:</Label>
-        <textarea
-          name="content"
-          id="content"
-          onChange={(e) => setContent(e.target.value)}
-          data-testid="content-text"
+    <section
+      className="flex flex-col space-y-6"
+      data-testid="admin-notifications-content"
+    >
+      <p>
+        Choose the users you would like to email in INSERT_CURRENT_LEAGUE_HERE
```

Comment:
nit: `INSERT_CURRENT_LEAGUE_HERE` I wonder if there's a better way to word this for brevity, like YOUR_LEAGUE

Reply:
This is a placeholder and should be replaced with the league the admin has currently selected. Something @alexappleget should add as I'm purely doing UI/UX on this.

Reply:
I actually did this myself, whoops /s